src/libraries/Microsoft.Extensions.Logging.EventSource/tests/Microsoft.Extensions.Logging.EventSource.Tests.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetFrameworkMinimum)</TargetFrameworks>
     <EnableDefaultItems>true</EnableDefaultItems>
     <EventSourceSupport Condition="'$(TestNativeAot)' == 'true'">true</EventSourceSupport>
+    <EnableNativeEventPipe Condition="'$(TestNativeAot)' == 'true'">true</EnableNativeEventPipe>
```

Comment:
Why isn't `EnableNativeEventPipe` defaulted to `true` if `EventSourceSupport=true`? According to the PR description, it is not a supported configuration to just set `EventSoureSupport=true` without the "Native AOT Library". So it would make sense to only require the dev to set `EventSoureSupport` and it "just work".

Reply:
We want to have the option to separate managed and native event support in NativeAOT (for size and other reasons). This is not fully flushed out in this PR though.

Reply:
I agree having the separate options make sense.

But forcing apps to set both doesn't make sense. An app should just need to set `EventSourceSupport=true`, and "the right thing" happens.

Reply:
Created https://github.com/dotnet/runtime/issues/82231 to track this

Reply:
#82200 changed the default so that EventPipe is enabled if EventSource is true